src/main/java/com/denizenscript/ddiscordbot/commands/DiscordChannelCommand.java

```diff
@@ -0,0 +1,149 @@
+package com.denizenscript.ddiscordbot.commands;
+
+import com.denizenscript.ddiscordbot.DenizenDiscordBot;
+import com.denizenscript.ddiscordbot.objects.DiscordChannelTag;
+import com.denizenscript.ddiscordbot.objects.DiscordGroupTag;
+import com.denizenscript.denizen.utilities.debugging.Debug;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.scripts.commands.Holdable;
+import net.dv8tion.jda.api.entities.Category;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.requests.restaction.ChannelAction;
+import org.bukkit.Bukkit;
+
+public class DiscordChannelCommand extends AbstractCommand implements Holdable {
+
+    public DiscordChannelCommand() {
+        setName("discordchannel");
+        setSyntax("discordchannel [id:<id>] (group:<group_id>) (channel:<channel_id>) [create/delete] (name:<name>) (description:<description>) (category:<category_id>)");
```

Comment:
This is two separate commands (create and delete). Or probably even a create command and a delete _mech_ considering delete takes effectively zero input.